index.bs

```diff
@@ -3818,29 +3893,34 @@ partial dictionary MLOpSupportLimits {
   </tr>
   <tr>
     <td>{{weight}}</td>
-    <td>[=/same as=] {{input}}</td>
+    <td>[=/same type as|same as=] {{input}}</td>
     <td>3</td>
   </tr>
   <tr>
     <td>{{recurrentWeight}}</td>
-    <td>[=/same as=] {{input}}</td>
+    <td>[=/same type as|same as=] {{input}}</td>
     <td>3</td>
   </tr>
   <tr>
     <td>{{MLGruOptions/bias}}</td>
-    <td>[=/same as=] {{input}}</td>
+    <td>[=/same type as|same as=] {{input}}</td>
     <td>2</td>
   </tr>
   <tr>
     <td>{{MLGruOptions/recurrentBias}}</td>
-    <td>[=/same as=] {{input}}</td>
+    <td>[=/same type as|same as=] {{input}}</td>
     <td>2</td>
   </tr>
   <tr>
     <td>{{MLGruOptions/initialHiddenState}}</td>
-    <td>[=/same as=] {{input}}</td>
+    <td>[=/same type as|same as=] {{input}}</td>
     <td>3</td>
   </tr>
+  <tr>
+    <td>*outputs*</td>
+    <td>[=/same type as|same as=] {{input}}</td>
+    <td>3 (and 4, if {{MLGruOptions/returnSequence}} is true)</td>
+  </tr>
```

Comment:
Can we split these into separate tensors for clarity? The "3 (and 4..." confused me :b.

```html
  <tr>
    <td>*outputs[0]*</td>
    <td>[=/same type as|same as=] {{input}}</td>
    <td>3 (and 4, if {{MLGruOptions/returnSequence}} is true)</td>
  </tr>
  <tr>
    <td>*outputs[1]* if {{MLGruOptions/returnSequence}} is true</td>
    <td>[=/same type as|same as=] {{input}}</td>
    <td>4</td>
  </tr>
```

Same for lstm and lstmCell.

Reply:
What do you think of this idea? Rest looks good to me.

Reply:
Done in 293e02a320fd3c2fd8129c3d4e487d36af0b1f1f - please take a look?

Reply:
Thanks 👍.